CONTRIBUTING.md

```diff
@@ -0,0 +1,36 @@
+Thanks for contributing. You rock!
+
+# Submitting an Issue
+
+All issues are tracked as part of the [cesium-ion-community](https://github.com/AnalyticalGraphicsInc/cesium-ion-community) repository.
+
+If you think you've found a bug, first search the [issues](https://github.com/AnalyticalGraphicsInc/cesium-ion-community/issues) there. If an issue already exists, please add a comment expressing your interest and any additional information. This helps us prioritize issue. If you are unable to find an existing issue, please create a new one with as much information needed for us to reproduce the problem or implement a feature.
+
+# Getting Started Contributing
+
+Everyone is welcome to contribute!
+
+See the [Developer Guide](Documentation/Contributors/DeveloperGuide/README.md) for how to set up your system for development.
+
+Always feel free to introduce yourself on the [issue tracker](https://github.com/AnalyticalGraphicsInc/cesium-ion-community/issues) to brainstorm ideas and ask for guidance.
+
+# Opening a Pull Request
+
+We love pull requests. We strive to promptly review them, provide feedback, and merge. Interest in Cesium is at an all-time high so the core team is busy. Following the tips in this guide will help your pull request get merged quickly.
+
+If you plan to make a major change, please start or comment on an [issue](https://github.com/AnalyticalGraphicsInc/cesium-ion-community/issues) about it first. Pull requests for small features and bug fixes can generally just be opened without discussion on the forum.
+
+## Contributor License Agreement (CLA)
+
+Before we can merge a pull request, we require a signed Contributor License Agreement. There is a CLA for:
+
+* [individuals](Documentation/Contributors/CLAs/individual-cla-agi-v1.0.txt) and
+* [corporations](Documentation/Contributors/CLAs/corporate-cla-agi-v1.0.txt).
+
+This only needs to be completed once, and enables contributions to all of the projects under the [Analytical Graphics Inc](https://github.com/AnalyticalGraphicsInc) organization, including CesiumJS. The CLA ensures you retain copyright to your contributions, and provides us the right to use, modify, and redistribute your contributions using the [Apache 2.0 License](LICENSE.md).
+
+Please email a completed CLA with all fields filled in to [cla@agi.com](mailto:cla@agi.com). Related questions are also welcome.
+
+# Code of Conduct
```

Comment:
Does this need a separate header with one sentence? I feel like it would have been appropriate to mention under the "getting started contributing" where you say all are welcome.

Reply:
I pared down the overall text further but I still think it's important to have this under a heading to call attention to it.